### PR TITLE
chore(web): replace all old icon method uses

### PR DESCRIPTION
**Pull request by TrygveUrdahl**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

### CODE REVIEW

File: packages/web/src/app/doc-pages/components/list-doc/list-icon-ceg/list-icon-ceg.component.html [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-      <i class="e-icon e-icon e-icon--han-color e-icon--md" aria-hidden="true"></i>
+      <e-icon name="hanColor" size="md"></e-icon>
     </span>
     <strong>HAN-port:</strong>
     For å åpne HAN-porten kan du logge inn på Min side og bestille dette.
   </li>
   <li>
     <span class="e-list__icon">
-      <i class="e-icon e-icon e-icon--powermeter_old e-icon--md" aria-hidden="true"></i>
+      <e-icon name="powermeterOld" size="md"></e-icon>
```
Comment (TomSchrier) [inline packages/web/src/app/doc-pages/components/list-doc/list-icon-ceg/list-icon-ceg.component.html:18]: Is setting the size to `md` necessarily here due to the list?

File: packages/web/src/app/doc-pages/components/popover-doc/popover-ceg/popover-ceg.component.html [hunk 2]
```diff
@@ -6,8 +6,8 @@
     aria-label="Popover trigger"
   >
     <span class="e-btn__icon">
-      <i class="e-icon e-icon--information_circle" aria-hidden="true"></i>
-      <i class="e-icon e-icon e-icon--information_circle-filled-color" aria-hidden="true"></i>
+      <e-icon name="informationCircle" size="md"></e-icon>
+      <e-icon name="informationCircleFilledColor" size="md"></e-icon>
```
Comment (TomSchrier) [inline packages/web/src/app/doc-pages/components/popover-doc/popover-ceg/popover-ceg.component.html:10]: Here too 👀

**Review by TomSchrier — APPROVED** [review]
LGTM 🎉

File: packages/web/src/app/doc-pages/components/list-doc/list-icon-ceg/list-icon-ceg.component.html (hunk 1 above)
Comment (TrygveUrdahl) [reply packages/web/src/app/doc-pages/components/list-doc/list-icon-ceg/list-icon-ceg.component.html:18]: Good catches! I was pretty much just blindly copying (oops), these aren't needed!

**Pull request merged**